indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -129,6 +145,7 @@ FormItem.defaultProps = {
   fieldIsPersonalData: false,
   fieldIsRequired: false,
   isRequired: false,
+  retentionPeriod: 0,
```

Comment:
I think we use `null` for no value, don't we? So maybe using `null` here would be better..